docs/codeql/CONTRIBUTING.MD

```diff
@@ -0,0 +1,18 @@
+# Contributing to CodeQL docs
+
+We welcome contributions to our CodeQL docs. Want to improve existing docs or add new information you think would be helpful? Then please go ahead and open a pull request!
+
+## Contributing to CodeQL docs on `codeql.github.com`
+
+To make changes to the documentation on [codeql.github.com](https://codeql.github.com/docs/codeql-overview/), you can make changes to the documentation files using the GitHub UI, a codespace, or a local text editor, and then open a pull request for review. For more information about the format and structure of the CodeQL documentation on [codeql.github.com](https://codeql.github.com/docs/codeql-overview/), please see the [README](https://github.com/github/codeql/blob/main/docs/codeql/README.rst).
+
+## Contributing to CodeQL CLI docs on `docs.github.com`
+
+We are in the process of moving all documentation about the CodeQL CLI from [codeql.github.com](https://codeql.github.com/docs/codeql-cli/) to the public [github/docs](https://github.com/github/docs) repository so that this documentation is published on the [GitHub Docs](https://docs.github.com/en/code-security/code-scanning) site. This includes all articles under "[Using the CodeQL CLI](https://codeql.github.com/docs/codeql-cli/using-the-codeql-cli/)" and "[CodeQL CLI reference](https://codeql.github.com/docs/codeql-cli/codeql-cli-reference/)" categories. This will make it easier for code scanning users to find information about using CodeQL to query their codebases.
```

Comment:
Just realized that I suggested mixed source/publication links in the first sentence here 🤦🏻‍♀️ 

Suggest we probably change this as follows:

```suggestion
We are in the process of moving all documentation about the CodeQL CLI from [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql) to the public [github/docs](https://github.com/github/docs) repository so that this documentation is published on the [GitHub Docs](https://docs.github.com/en/code-security/code-scanning) site. This includes all articles that are currently published under "[Using the CodeQL CLI](https://codeql.github.com/docs/codeql-cli/using-the-codeql-cli/)" and "[CodeQL CLI reference](https://codeql.github.com/docs/codeql-cli/codeql-cli-reference/)" categories on the CodeQL microsite. This will make it easier for code scanning users to find information about using CodeQL to query their codebases.
```